pkg/stats/collector.go

```diff
@@ -405,9 +405,15 @@ func (s *BufferedCollector) CollectCommPrefs(email, installationID string, featu
 	}
 	ctx := context.Background()
 	err := s.sender.UpdateCommPrefs(ctx, commPrefs)
+	status := "success"
 	if err != nil {
 		s.log.WithError(err).Info("could not update comm prefs")
+		status = "failed"
 	}
+	s.CollectEvent(Event{
+		Class: "global",
+		Name:  fmt.Sprintf("update_comm_prefs_%s", status),
+	})
```

Comment:
Suggestion - it will be easy to lookup later
```suggestion
	name := "update_comm_prefs_success"
	if err != nil {
		name = "update_comm_prefs_failed"
		s.log.WithError(err).Info("could not update comm prefs")
	}
	s.CollectEvent(Event{
		Class: "global",
		Name:  name,
	})
```